l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -520,6 +521,35 @@ def invoiceCreate(
             )
         credit_account_id = purchase_journal.default_credit_account_id.id
         invoice_lines = []
+        comment = ''
+        #2.1.1
+        docType_id = False
+        invtype = 'in_invoice'
+        docType = FatturaBody.DatiGenerali.DatiGeneraliDocumento.TipoDocumento
+        if docType:
+            docType_ids = ftpa_doctype_poll.search(
+                cr, uid,
+                [
+                    ('code', '=', docType)
+                ],
+                context=context
+            )
+            if docType_ids:
+                docType_id = docType_ids[0]
+            else:
+                raise orm.except_orm(
+                    _("Error"),
+                    _("tipoDocumento %s not handled")
+                    % docType)
+            if docType == 'TD04':
+                invtype = 'in_refund'
+            elif docType == 'TD05':
+                invtype = 'out_refund'
```

Comment:
type = out_refund in openerp sono le note di credito emesse. il modulo fatturapa_in crea fatture ricevute e note di credito ricevute